docs/architecture.md

```diff
@@ -8,7 +8,7 @@ TIDES consists of a package of files as defined in the following table.
 
 **The following table is automatically generated from [`tides.spec.json`](http://github.com/TIDES-transit/blob/main/spec/tides.spec.json)**
```

Comment:
This link is missing the repo name, should be https://github.com/TIDES-transit/TIDES/blob/main/spec/tides.spec.json